common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -95,7 +95,8 @@ def test_bulk_email_a11y(self):
         ])
         self.send_email_page.a11y_audit.config.set_rules({
             "ignore": [
-                'button-name',  # TODO: AC-491
+                'button-name',  # TinyMCE button is missing accessible text
+                'list',  # TODO: AC-491,
```

Comment:
My guess is the comma? I didn't add anything new. I just re-added `button-name` here, with a new comment. Let me double check.